src/code/z_scene_table.c

```diff
@@ -2408,12 +2408,14 @@ void func_8009FE58(GlobalContext* globalCtx) {
         D_8012A3A0 += 1820;
 
         temp = 0.020000001f;
-        func_800AA76C(&globalCtx->view, ((360.00018f / 65535.0f) * (M_PI / 180.0f)) * temp * Math_CosS(D_8012A39C),
-                      ((360.00018f / 65535.0f) * (M_PI / 180.0f)) * temp * Math_SinS(D_8012A39C),
-                      ((360.00018f / 65535.0f) * (M_PI / 180.0f)) * temp * Math_SinS(D_8012A3A0));
-        func_800AA78C(&globalCtx->view, 1.f + (0.79999995f * temp * Math_SinS(D_8012A3A0)),
-                      1.f + (0.39999998f * temp * Math_CosS(D_8012A3A0)), 1.f + (1 * temp * Math_CosS(D_8012A39C)));
-        func_800AA7AC(&globalCtx->view, 0.95f);
+        View_SetDistortionDirRot(&globalCtx->view,
+                                 ((360.00018f / 65535.0f) * (M_PI / 180.0f)) * temp * Math_CosS(D_8012A39C),
+                                 ((360.00018f / 65535.0f) * (M_PI / 180.0f)) * temp * Math_SinS(D_8012A39C),
+                                 ((360.00018f / 65535.0f) * (M_PI / 180.0f)) * temp * Math_SinS(D_8012A3A0));
```

Comment:
"direction rotation" doesnt really make sense in my head for what it would be, while "rotation direction" sounds a bit better. i dont know enough about the implementation to see if this is what you actually mean, does that name still make sense

Reply:
I might be fine reverting back to `distortionRot`, I think the original issue was that this is not applying a rotation to the view (i.e. through the projection matrix). But instead, this is applying a rotation to the distortion itself i.e. orienting the distortion in a particular way. Isn't that captured in the name `distortionRot`? 

If it were to apply a rotation to the view (which it's not), it would be called `viewRot` or `projectionRot`. 

@Dragorn421 thoughts?

Reply:
my thoughts are on Discord https://discord.com/channels/688807550715560050/688851451014938642/931683801216725032
I @'ed fig but I guess we flooded right after so maybe he didn't see

> I think the concern with naming the distortion direction rotation like `distortRot` is it may be read as a rotational distort, which it isn't
> and `distortDir` would also be confusing because a direction is usually a vector ("arrow pointing somewhere"), not a rotation
> so the clever middle ground of `DirRot` was coined :5Head: 

I also linked the two demo vids on Discord to show this affects the direction of the distortion, and isn't a rotational distortion

Reply:
What about `distortionOrientation`? A little on the verbose side but it can't be confused with a distortion that rotates the screen

Reply:
if that works as a name i would prefer it to avoid the confusion
and sorry for not getting back to this, got distracted by other things lol